praetorian_cli/handlers/add.py

```
@@ -5,11 +5,32 @@
+        # Return a dynamic command that adds entities with the label=cmd_name
+        @click.command(name=cmd_name)
+        @click.option('-f', '--field', 'field_list', multiple=True,
+                      help='Field in format name:value (can be specified multiple times)')
+        @cli_handler
+        def _dynamic(chariot, field_list):
+            try:
+                dynamic_fields = parse_name_value_fields(field_list)
+                if dynamic_fields is None:
+                    return
+                print_json(chariot.generic.add(cmd_name, dynamic_fields))
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)